docs/wiki/file_manipulation.md

```diff
@@ -0,0 +1,528 @@
+
+@:choice(syntax)
+
+```scala 3 mdoc:compile-only
+withTempDirectory: dir => 
+  (dir / "its_going_to_go_soon.mp3").createFile
+```
+
+@:choice(static)
+
+```scala 3 mdoc:compile-only
+FilesOs.withTempDirectory: dir =>
+  FilesOs.createFile(dir / "its_going_to_go_soon.mp3")
+```
+
+@:choice(fs2)
+
+```scala mdoc:compile-only
+Files[IO].tempDirectory.use: dir => 
+  Files[IO].createFile(dir / "its_going_to_go_soon.mp3")
+
+```
+
+@:@
+
+### `createSymbolicLink`
+
+Creates a [Symbolic Link](https://en.wikipedia.org/wiki/Symbolic_link) to a file. Requires the destination of the symlink and the path of the target file to link, and optionally some permissions:
+
+@:select(api-style)
+
+@:choice(syntax)
+
+```scala mdoc:compile-only
+val linkPath   = Path("store/the/link/here/symlink")
+val targetPath = Path("path/to/file/to/target.sh")
+
+linkPath.createSymbolicLink(targetPath)
+```
+
+@:choice(static)
+
+```scala mdoc:compile-only
+val linkPath   = Path("store/the/link/here/symlink")
+val targetPath = Path("path/to/file/to/target.sh")
+
+FilesOs.createSymbolicLink(linkPath, targetPath)
+```
+
+@:choice(fs2)
+
+```scala mdoc:compile-only
+val linkPath   = Path("store/the/link/here/symlink")
+val targetPath = Path("path/to/file/to/target.sh")
+
+Files[IO].createSymbolicLink(linkPath, targetPath)
+```
+
+@:@
+
+## Deleting files and directories
+
+### `delete`
+
+Deletes a file or empty directory that must exist (otherwise it will fail). 
+
+@:select(api-style)
+
+@:choice(syntax)
+
+```scala mdoc:compile-only
+path.createFile >> 
+  path.write("TOP SECRET 🚫, MUST DELETE") >>
+    path.delete
+```
+
+@:choice(static)
+
+```scala mdoc:compile-only
+FilesOs.createFile(path) >>
+  FilesOs.write(path, "TOP SECRET 🚫, MUST DELETE") >>
+    FilesOs.delete(path)
+```
+
+@:choice(fs2)
+
+```scala mdoc:compile-only
+for
+  _ <- Files[IO].createFile(path)
+
+  _ <- Stream.emit("TOP SECRET 🚫, MUST DELETE")
+        .through(Files[IO].writeUtf8(path))
+        .compile
+        .drain
+
+  _ <- Files[IO].delete(path)
+yield ()
+```
+
+@:@
+
+### `deleteIfExists`
+
+Similar to `delete`, but returns `true` if the deletion was successful instead of failing: 
+
+@:select(api-style)
+
+@:choice(syntax)
+
+```scala mdoc:compile-only
+path.deleteIfExists >>= 
+  (deleted => IO.println(s"Was the file deleted? $deleted"))
+```
+
+@:choice(static)
+
+```scala mdoc:compile-only
+FilesOs.deleteIfExists(path) >>= 
+  (deleted => IO.println(s"Was the file deleted? $deleted"))
+```
+
+@:choice(fs2)
+
+```scala mdoc:compile-only
+Files[IO].deleteIfExists(path) >>= 
+  (deleted => IO.println(s"Was the file deleted? $deleted"))
+```
+
+@:@
+
+### `deleteRecursively`
+
+With the previous functions, the directory had to be empty to be deleted. The difference with this method is that it recursively deletes all files or folders contained inside it, optionally following symbolic links if specified: 
+
+Recursively deletes all files or folders, following symbolic links if specified. 
```

Comment:
Doesn't this line repeat the previous one?

Reply:
You are right >:)